src/usdb_syncer/resource_dl.py

```diff
@@ -96,7 +97,14 @@ def download_video(
             logger.debug(f"error downloading video url: {url}")
             return None
 
-    return ext or os.path.splitext(filename)[1][1:]
+    actual_file_ext = ext or os.path.splitext(filename)[1][1:]
+    path = f"{path_stem}.{actual_file_ext}"
+    if not (filetype.is_video(path) or filetype.is_audio(path)):
```

Comment:
Is this necessary? I can't imagine yt_dlp works with anything but valid video URLs.

Reply:
to be honest I just put the image check in the image download function and did the same for audio/video. But it may be useful if yt_dlp left an empty file after an exception or video/audio post processing had an error and ceated a wrong formatted / empty file.

Reply:
But your code isn't even executed in case of an exception. And even if you moved it into the except block and yt_dlp did leave any files behind, you wouldn't know which path to check.
In summary, cleaning up is really yt_dlp's responsibility here and it seems to do that very reliably.

Reply:
oh true, did not register the return in the except block. I agree and will remove the check here